<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sweets</class>
 <widget class="QDialog" name="sweets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сладкое</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Шоколад молочный</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Шоколад темный</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Яйцо шоколадное</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Паста ореховая</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Торт красный бархат</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Торт медовик</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Торт Прага</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
